src/main/java/seedu/apollo/Ui.java

```diff
@@ -102,6 +102,23 @@ public void printModuleList(ArrayList<Module> allModules) {
             System.out.println(i+1 + "." + allModules.get(i).toString());
         }
     }
+    /**
+     * For {@code delmod} command.
+     * Prints warning to ask if user is sure that whether they want to delete that module from their moduleList
+     */
+    public void printModuleDeleteWarningMessage() {
+        System.out.println("Are you sure you want to remove this module from your Module list? (Y/N)");
+
+    }
```

Comment:
Not sure if we should do multistep commands (according to the website, it says to avoid it)